simpleflow/task.py

```diff
@@ -62,6 +66,12 @@ def __init__(self, activity, *args, **kwargs):
             raise TypeError(
                 "Wrong value for `activity`, got {} instead".format(type(activity))
             )
+
+        self.pre_execute_funcs = []
+        self.post_execute_funcs = []
+        # Avoid spreading middlewares in kwargs
+        self.__load_middlewares__(kwargs.pop("simpleflow_middlewares", None))
```

Comment:
```suggestion
        self.load_middlewares(kwargs.pop("simpleflow_middlewares", None))
```
This isn't a Python special method :slightly_smiling_face: